src/main/java/dabang/star/cafe/domain/admin/OptionAdminServiceImpl.java

```diff
@@ -50,7 +50,7 @@ public void updateOption(Option option) {
     private Option loadById(int optionId) {
 
         return optionRepository.findById(optionId).orElseThrow(
-                () -> new OptionNotFoundException("option not found")
+                () -> new OptionNotFoundException("option not found by id")
```

Comment:
아 이런 의도였는데 잘못 전달되었나보네요~ 

```suggestion
                () -> new OptionNotFoundException("option not found by id : " + optionId)
```